packages/query-core/src/__tests__/streamedQuery.test.tsx

```diff
@@ -391,13 +391,18 @@ describe('streamedQuery', () => {
     })
   })
 
-  test('should support maxChunks', async () => {
+  test('should support custom reducer', async () => {
     const key = queryKey()
+
     const observer = new QueryObserver(queryClient, {
       queryKey: key,
-      queryFn: streamedQuery({
-        queryFn: () => createAsyncNumberGenerator(3),
-        maxChunks: 2,
+      queryFn: streamedQuery<number, Record<number, boolean>>({
```

Comment:
let’s not pass types this way - as it won’t test if type inference works. Instead, let’s type the initialValue:

```
initialValue: {} as Record<number, boolean>
```